vignettes/calibration-2021.Rmd

```diff
@@ -0,0 +1,292 @@
+GrandTab includes both hatchery and natural spawners, we will scale GrandTab escapement
+values using an estimated proportion natural spawners for each watershed.
+This will allow us to compare predicted natural spawners to psuedo GrandTab natural spawners.
+
+For calibration, we have prepared the GrandTab data in two ways:
+
+1. `DSMCalibrationData::grandtab_observed` - This data will be used in the fitness function to measure the difference between model predictions and observed escapement. Missing values are NA and we made all records less than 100 NA to account for a lack of confidence for counts less than 100. 
+
+2. `DSMCalibrationData::grandtab_imputed` - This data will be used to calculate the number of juveniles during the 20 year simulation. The GrandTab data is incomplete for many watersheds during the 20 year period of calibration. For watersheds with no GrandTab data, we used 40 as the default escapement value. For watersheds with incomplete data, we used the mean escapement value. 
+
+**Observed Values**
+
+```{r, echo=FALSE, warning=FALSE, fig.fullwidth=TRUE, fig.width=8, fig.height=4}
+as_tibble(grandtab_observed$winter[keep, ]) %>% 
+  mutate(watershed = winterRunDSM::watershed_labels[keep]) %>% 
+  gather(year, spawners, -watershed) %>% 
+  mutate(year = readr::parse_number(year)) %>% 
+  plot_ly(x = ~year, y = ~spawners, type = "scatter", mode = "lines", 
+          color = ~watershed, 
+          hoverinfo = "text", 
+          text = ~paste("<b>Year</b>", year, "<br><b>Spawners</b>", 
+                        winterRunDSM::pretty_num(spawners))) %>% 
+  config(displayModeBar = FALSE)
+
+```
+
+
+
+**Imputed Values**
+
+```{r, echo=FALSE, warning=FALSE, fig.fullwidth=TRUE, fig.width=8, fig.height=4}
+
+as_tibble(grandtab_imputed$winter[keep, ]) %>% 
+  mutate(watershed = winterRunDSM::watershed_labels[keep]) %>% 
+  gather(year, spawners, -watershed) %>% 
+  mutate(year = readr::parse_number(year)) %>% 
+  plot_ly(x = ~year, y = ~spawners, type = "scatter", mode = "lines", 
+          color = ~watershed, 
+          hoverinfo = "text", 
+          text = ~paste("<b>Year</b>", year, "<br><b>Spawners</b>", 
+                        winterRunDSM::pretty_num(spawners))) %>% 
+  config(displayModeBar = FALSE)
+```
+
+### Selected Proxy Years
+
+The data inputs to the DSM are for years 1980-1999. We selected proxy years for 1998-2017
+from the 1980-1999 model inputs by [comparing the DWR water year indices](https://cdec.water.ca.gov/reportapp/javareports?name=WSIHIST) in this [analysis](https://github.com/CVPIA-OSC/DSMCalibrationData/blob/main/data-raw/synthetic_years.R).
+
+These inputs are generated by calling `DSMCalibrationData::set_synth_years(params)`, with a `params` object from the life cycle model ex: `fallRunDSM::params`. See the documentation for more details on the [DSMCalibrationData](https://github.com/CVPIA-OSC/DSMCalibrationData)
```

Comment:
change the reference to fallRunDSM:: to winterRunDSM::

Reply:
fixed